package.json

```diff
@@ -110,8 +110,9 @@
     "style-loader": "0.18.2",
     "watch-http-server": "0.7.6",
     "webpack": "3.4.1",
+    "webpack-cli": "3.2.1",
```

Comment:
why is it necessary?

Reply:
cant build new webpack since they split it to 3 packages (`wepack`, `webpack-cli` and `webpack-dev-server`)